package.json

```diff
@@ -9,7 +9,7 @@
   },
   "scripts": {
     "start": "nx serve",
-    "build": "nx build",
+    "build": "nx run-many -t build",
```

Comment:
this can be reverted..the intention here is to use as a shortcut to run the specific project name

for eg.

```
pnpm build history-utility
```

where the alternative is

```
pnpm nx build history-utility
```

Reply:
would remove the coedesandbox build config as we arent really using it

Reply:
> this can be reverted..the intention here is to use as a shortcut to run the specific project name
> 
> for eg.
> 
> ```
> pnpm build history-utility
> ```
> 
> where the alternative is
> 
> ```
> pnpm nx build history-utility
> ```
 
yea thought the action was sanity checking external pr's by building everything